src/main/java/bio/terra/cli/businessobject/resource/AwsS3StorageFolder.java

```diff
@@ -121,6 +124,34 @@ public static String resolve(
     return includeUrlPrefix ? AWS_S3_BUCKET_URL_PREFIX + resolvedPath : resolvedPath;
   }
 
+  public JSONObject getCredentials(
+      CredentialsAccessScope scope, int duration, boolean includeConsoleLink) {
+    // call WSM to get credentials
+    AwsCredential awsCredential =
+        WorkspaceManagerService.fromContext()
+            .getAwsS3StorageFolderCredential(
+                Context.requireWorkspace().getUuid(),
+                id,
+                scope == CredentialsAccessScope.READ_ONLY
+                    ? AwsCredentialAccessScope.READ_ONLY
+                    : AwsCredentialAccessScope.WRITE_READ,
+                duration);
+
+    JSONObject object = new JSONObject();
+    object.put("AccessKeyId", awsCredential.getAccessKeyId());
```

Comment:
Add version

Reply:
Hard coded to 1 in WSM schema object https://github.com/DataBiosphere/terra-workspace-manager/pull/1215
this keeps CLI and wsm responses consistent (and same)